app/experimenter/nimbus-ui/src/hooks/useChangeOperationMutation.test.tsx

```diff
@@ -113,24 +100,13 @@ const setupTestHook = (refetch?: () => Promise<unknown>) => {
     );
   }, []);
 
-  const { result, waitForNextUpdate } = renderHook(
+  return renderHook(
     () => useChangeOperationMutation(experiment, refetch, ...mutationSets),
     {
       wrapper,
       initialProps: { mocks },
     },
   );
-
-  const invokeCallback = (callbackIdx = 0) =>
-    setTimeout(() => result.current.callbacks[callbackIdx](), 0.1);
```

Comment:
I'm guessing this caused bad vibes. After re-running this test over & over locally, I saw that every now and then the entire hook would complete its run before the first `await waitForNextUpdate()` in the test.

Read through the docs some more. Realized I could just await the hook's complete run, then check [`result.all`](https://react-hooks-testing-library.com/reference/api#result) afterward to synchronously assert what should have happened.